demos/risk/InfrastructureRisk/RoadNetwork/README.md

```diff
@@ -0,0 +1,44 @@
+Road Network Demo
+-----------------
+
+In this example, we present a grid-like road network with 29 nodes and 38
+edges. The exposure model is prepared for both node- and edge-like components.
+The road network consists of road segments, bridges, connections and TAZ
+(Traffic Analysis Zone) nodes. The information of the `start_node` and
+`end_node` of each edge indicated how components are interlinked. With all the
+attributes from the exposure model, the network is created.
+
+In the `expo_demo_edges`, the location is for the centroid of the edges. The
+weights refer to the lengths and thus the graph is weighted.
+
+As seen in the `expo_demo_nodes`, we have the nodes acting as TAZ. Therefore,
+partial connectivity loss, weighted connectivity loss and efficiency loss are
+calculated.
+
+Note: this is an example for the demonstrative purpose and does not represent a
+real network.
+
+**Hazard**
+
+Expected runtime: 5 seconds
+
+Outputs:
+
+- Events
+- Ground Motion Fields
+
+**Risk**
+
+Expected runtime: 12 seconds
+
```

Comment:
I would measure the times on a a windows machine, since most users will have that, and I expect 12 seconds of a mac with M1 processor to be over 1 minute on a Windows machine.

Reply:
Done